source/classification1.md

```diff
@@ -1988,13 +1855,17 @@ prediction_plot = (
     .encode(
         x=alt.X("Area"),
         y=alt.Y("Smoothness"),
-        color=alt.Color("Class", scale=alt.Scale(range=colors), title="Diagnosis"),
+        color=alt.Color("Class", title="Diagnosis"),
     )
 )
-
 unscaled_plot + prediction_plot
 ```
 
+```{code-cell} ipython3
+:tags: [remove-input]
+glue("fig:05-upsample-2", (unscaled_plot + prediction_plot))
```

Comment:
```suggestion
glue("fig:05-upsample-3", (unscaled_plot + prediction_plot))
```

This overwrites a figure with the same name above. Maybe the figure name below should be changed too and use this one? Or where is this figure meant to be used?